packages/jest-types/__typetests__/jest.test.ts

```diff
@@ -183,6 +184,9 @@ expect(jest.setMock('moduleName')).type.toRaiseError();
 expect(jest.unmock('moduleName')).type.toEqual<typeof jest>();
 expect(jest.unmock()).type.toRaiseError();
 
+expect(jest.unstable_unmockModule('moduleName')).type.toEqual<typeof jest>();
```

Comment:
Thanks. The type test looks good.

Some matchers got renamed in latest version of TSTyche (the type testing tool). Both `toEqual` and `toBe` do the same job, but would be better to use the new name here (I just opened an upgrade PR #15088):

```suggestion
expect(jest.unstable_unmockModule('moduleName')).type.toBe<typeof jest>();
```